### PR TITLE
Farm page functionality

### PR DESCRIPTION
**Pull request by Kmittleider**
Update city, state to display correctly on farm cardviews.
Each farm card can now be clicked to lead to that specific farm's page.
Change the text color on product page and producer page from red to dark grey.
**Farm Page now works as intended**

### CODE REVIEW

**Review by sohumgala — APPROVED** [review]
looks good on my end, I'd wait for another frontend review before merging to make sure everything is good

**Review by agul9 — APPROVED** [review]
looks good, great work Katherine! just need to add the bottom navigation!

**Pull request merged**
